data/projects/create_jenkins_plugin.md

```diff
@@ -1,26 +1,45 @@
-Jenkins users should be able to see coala results right in the 
-Jenkins UI. The result of this project should be a full fledged Jenkins 
-integration, living in an own repository with a full testsuite.
+Jenkins is the most popular ci-cd server, and there are many corporations 
+who already use it. Integrating coala into Jenkins can attract business 
+clients better who could end up sponsoring/helping out coala. It also generally allows a wider range of users to use coala in their prefered way.
 
-If you find this fun you'll be the one knowing this component best and will 
-get the chance to maintain it on your own if you want - always with the
-support of the community guarding your back of course.
+[Writing a coala Jenkins Plugin - Google Groups](https://groups.google.com/forum/#!msg/jenkinsci-dev/2LciAQgdRcY/Kv2rQlwXAAAJ) 
+could help you to build the plugin. 
+
+There is a similiar plugin in Jenkins plugin repository, 
+[warnings plugin](https://wiki.jenkins-ci.org/display/JENKINS/Warnings+Plugin). 
+A parser for warnings plugin could be implemented in the early 
+stage of developing. It's easy to get started.
+
+#### Final Goal
+
+The goal of this project is to offer a coala plugin for Jenkins to 
```

Comment:
Could this goal explain further about what a coala jenkin plugin could do? Is it to make installing coala and it's configuration easier in jenkin workflow then could output the result in nice way inside jenkin console?